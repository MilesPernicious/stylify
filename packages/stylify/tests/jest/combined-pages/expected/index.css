.s0{
	font-size:123px
}
.s1{
	line-height:123px
}
.s2{
	color:#fff
}
.s3{
	color:#888
}
.s4{
	font-size:456px
}
.s5{
	line-height:456px
}
.s6{
	color:yellow
}
.s7{
	font-family:arial
}
.s8{
	font-size:12px
}
.s9{
	text-align:center
}
.s10{
	font-size:48px
}
.s11{
	background:steelblue
}
.s12{
	max-width:320px
}
.s13{
	margin:0 auto
}
.s14{
	font-size:14px
}
.s15{
	background:black
}
.s16{
	color:darkred
}