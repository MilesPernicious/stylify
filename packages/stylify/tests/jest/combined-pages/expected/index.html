<!-- stylify-pregenerate font-size:123px line-height:123px color:#fff color:#888 /stylify-pregenerate -->
<!--
stylify-screens
	'testScreen': '(min-width: 123px)',
	'dynamic\\w+': (screen) => `(max-width: ${screen.replace('dynamic', '')})`
/stylify-screens

stylify-pregenerate font-size:456px line-height:456px color:yellow /stylify-pregenerate
-->

<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Document</title>
</head>
<body class="_1go3 _h0fu6">
	<div class="
		_1vegb8
		_h3cr9
		_dlen7
		_9gav2
		_m0vnad
		_olqrj2
		_bbjh7
		_i9e8r0
	">
		Index Page
	</div>
</body>
</html>
